locale/en/corteza-webapp-compose/block.yaml

```diff
@@ -449,6 +449,7 @@ recordList:
     withPresortedRecords: with presorted ({{0}}) record
   record:
     draggable: Can drag & drop records to order them
+    editMode: Users will be able to open record in edit mode
```

Comment:
This should be changed, since thats not what it does. It opens the record in edit mode when its clicked